mautic/mautic
# Add missing translation string

This should close #8492 and affects both m2 & m3.

| Q  | A
| --- | ---
| Bug fix? | x
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | resolves #8492 
| BC breaks? | 
| Deprecations? | 

#### Description:
Fixes small language string missing issue which has been bugging me for ages! I think this is the place it's missing, as it's in the lead bundle, but not in this bundle - whereas the preferred locale string is, which works fine.  It seems it's not getting added to the _lead_fields table due to being missing from this file.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Edit a contact and notice the missing language string which is rendered as mautic.lead.field.timezone in the default text

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com/8588)
2. Edit a contact and check to see if the correct language string is rendered
